src/NuGet.Core/NuGet.Build.Tasks/Common/MSBuildLogger.cs

```diff
@@ -11,7 +11,12 @@ namespace NuGet.Build
     /// <summary>
     /// TaskLoggingHelper -> ILogger
     /// </summary>
-    public class MSBuildLogger : LoggerBase, Common.ILogger
+#if PACK_TASKS
+    internal
+#else
+    public
+#endif
```

Comment:
Why we need to do this? IS there a conflict with `MSBuildLogger` class when running pack from msbuild?